doc/adr/0008-TimeRangeData-widget.md

```diff
@@ -0,0 +1,53 @@
+# 8. Time Range Data Widget
+
+Date: 2023-1-10
+
+## Status
+
+New
+
+## Context
+
+There has been a request for the ability for users to be able to select part of a trace by copy/pasting timestamp data.  An attempt was made to implement this feature, but a consensus could not be reached on the UI/UX.  Design needs to be agreed upon before any further implementation.
+
+The proposed design is subject to change based on feedback.
+
+### Proposed UX Design
+
+#### UX Flow
+
+A "Time Range Data" widget will be added to the left panel.
+
+![current](0008/widget-sidebar.png)
+
+The widget will display: View Start, View End, Selection Start, Selection End, and an Edit button.
```

Comment:
Do we need the edit button?

Reply:
Same opinion here, from the perspective of a user, I would prefer to do/navigate less to perform a task.

Reply:
I think having the editable numbers live in a permanent input box would fix this.

Reply:
> Do we need the edit button?



> Same opinion here, from the perspective of a user, I would prefer to do/navigate less to perform a task.


I can fix up an updated wireframe for this.  Input box will be visible for all situations, and simply clicking on it begins the editing process.

Reply:
> I can fix up an updated wireframe for this. Input box will be visible for all situations, and simply clicking on it begins the editing process.

Sounds good.

Reply:
Agreed. The last thing to do would be to remove the "and an Edit button" from the sentence above.